app/upgrades/v07/upgrades.go

```diff
@@ -0,0 +1,34 @@
+package v07
+
+import (
+	wasmkeeper "github.com/CosmWasm/wasmd/x/wasm/keeper"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+
+	poekeeper "github.com/confio/tgrade/x/poe/keeper"
+	poetypes "github.com/confio/tgrade/x/poe/types"
+	twasmkeeper "github.com/confio/tgrade/x/twasm/keeper"
+)
+
+func CreateUpgradeHandler(
```

Comment:
Looks good. And after this, the normal vote process can update the factory code uploaded.

This is indeed the simplest functioning version